### DISCUSSION TITLE
Please someone add spark new branch

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#23627] [General]
https://github.com/Spark-Rom/manifest

**Comment by Apon77** [comment]
do proper pr, i can merge

**Reply by OmegaBeam69** [reply]
https://github.com/ROM-builders/temporary/pull/23842

**Reply by OmegaBeam69** [reply]
https://github.com/ROM-builders/temporary/pull/23848
Sir new pr
